src/main/python/interfaces/widgets/label.py

```diff
@@ -41,48 +43,31 @@ def __init__(
         align and self.setAlignment(align)
 
     def reset(self):
+        """
+        Clean up text within label
+        :return: this function doesn't return value
```

Comment:
which one you mentioned?

Reply:
What I meant is that if a function doesn't return anything, we don't need to mention it in the docstring, and could make a lot of these docstrings one liners instead of multiline.  PEP is a little vague on this:

"The docstring for a function or method should summarize its behavior and document its arguments, **return value(s)**, side effects, exceptions raised, and restrictions on when it can be called **(all if applicable)**. Optional arguments should be indicated. It should be documented whether keyword arguments are part of the interface." (emphasis mine) https://www.python.org/dev/peps/pep-0257/

but it looks like we don't have to if there's no return value, which can make these comments more compact and easier to read

Reply:
Fixed